content/en/_index.md

```diff
@@ -250,6 +343,14 @@ try (CloudClient dlp = CloudClient.create()) {
   // make a request with the client
 }
 {{< /tab >}}
+{{< tab header="Ruby" >}}
+    require "google/cloud/spanner"
+    
+    # Initialize client that will be used to send requests. This client only needs to be created
+    # once, and can be reused for multiple requests. After completing all of your requests, call
+    # the "close" method on the client to safely clean up any remaining background resources,
```

Comment:
Does the Ruby spanner client have a "close" method?

Reply:
yes spanner client has a close method and I have updated the sample to show how to call it 👍